trieste/models/gpflow/sampler.py

```diff
@@ -795,24 +798,32 @@ def __init__(
         :param n_components: The desired number of features.
         """
 
+        super().__init__(model, n_components)
+
         if isinstance(model, SupportsGetInducingVariables):
-            inducing_points = model.get_inducing_variables()[0]  # [M, D]
+            self._inducing_points = model.get_inducing_variables()[0]  # [M, D]
```

Comment:
why assigning to self?

Reply:
Just to make explicit that we keep a handle to it, so it's not essential. Before it was indirect/hidden via self._canonical_feature_functions lambda. I also found it useful for debugging as I could more easily look at its state.